src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/interfaces/ReportDocPartial.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.reports.interfaces;
+
+import uk.gov.hmcts.ecm.common.model.listing.ListingData;
+
+public interface ReportDocPartial {
```

Comment:
Are you actually using this interface anywhere other than a class implementing it? What purpose does it serve?

Reply:
My thinking is that other report types should be refactored by implementing this interface. We are doing the template content building in one class now. At least, each report type can build & provide their specialised content. Right now I am using it in the current report. But I will use it for, say Claims Served, Members Sessions(which I work on next), etc.

Reply:
But you're not using the interface other than a class implements it. If that is the only usage then it provides no benefit. There's no point adding some partial code of a potential change